.vscode/c_cpp_properties.json

```diff
@@ -34,6 +34,44 @@
                     "${workspaceFolder}\\src"
                 ]
             }
+        },
+        {
+            "name": "Particle_nix",
+            "includePath": [
+                "${workspaceFolder}/src",
+                "~/.particle/toolchains/deviceOS/5.3.0/user/inc",
+                "/usr/include/",
+                "/usr/include/x86_64-linux-gnu/bits"
+            ],
+            "defines": [
+                "_DEBUG",
+                "UNICODE",
+                "_UNICODE"
```

Comment:
You are missing `TEST_VERSION` and `GTESTING=1` to be the same as your `CMakeLists.txt`